usaspending_api/api_contracts/contracts/award/idv/Download.md

```diff
@@ -0,0 +1,34 @@
+FORMAT: 1A
+HOST: https://api.usaspending.gov
+
+# IDV Download [/api/v2/download/idvs/]
+
+This end point returns a zipped file containing IDV data.
+
+## POST
+
++ Request (application/json)
+    + Parameters
```

Comment:
I can't get Dredd to understand this with "Parameters" here.  Needs to be "Attributes".

Reply:
Yep, `Parameters` works for GET requests; `Attributes` for POST